Source/Core/GeometryPipeline.js

```diff
@@ -2241,29 +2273,31 @@ define([
                 }
 
                 var offsetPoint = Cartesian3.add(intersection, offset, cartesian3Scratch6);
-                p0Attributes.position.values.push(p0.x, p0.y, p0.z, p1.x, p1.y, p1.z);
+                p0Attributes.position.values.push(p0.x, p0.y, p0.z, p0.x, p0.y, p0.z);
                 p0Attributes.position.values.push(offsetPoint.x, offsetPoint.y, offsetPoint.z);
                 p0Attributes.position.values.push(offsetPoint.x, offsetPoint.y, offsetPoint.z);
 
+                for (j = i0 * 3; j < i0 * 3 + 2 * 3; ++j) {
```

Comment:
Unroll this loop like you did for the other ones.  
It can be replaced by two lines of code.